docs/user-guide/release-notes/master.rst

```diff
@@ -55,6 +55,27 @@ Known Issues
 Upgrade Instructions for 2.6.x --> master
 -----------------------------------------
 
+Upgrade the packages using:
+
+::
+
+    sudo yum update
+
+After yum completes you should migrate the database using:
+
+::
+
+    sudo -u apache pulp-manage-db
+
+After migrating the database reload the systemd process using:
+
+::
+
+    sudo systemctl daemon-reload
```

Comment:
https://github.com/pulp/pulp/commit/e6ea3131796df60de3f7e3b98acc2ef108f517f7

The changes to the manage workers scripts require the reloading of systemd.